model/model_test.go

```diff
@@ -1324,6 +1331,144 @@ func testCron(t *testing.T, kind string) {
 	}
 }
 
+// testSubscriber tests Subscriber methods.
+func testSubscriber(t *testing.T, kind string) {
+	ctx := context.Background()
+	store, err := datastore.NewStore(ctx, kind, "vidgrind", "")
+	if err != nil {
+		t.Fatalf("could not create new store: %v", err)
```

Comment:
Fatal _should_ be used here because you can't really do much else if you can't create the store, but with how ```TestNetreceiverFileAccess``` and ```TestNetreceiverCloudAccess``` call the subtests, it means that all subtest calls after a Fatal call won't execute. 

I think either each subtest needs to have its own test, or we actually use subtests i.e. call t.Run() for each test (https://pkg.go.dev/testing#T.Run). It may look something like this:
```
func getFuncName(i interface{}) string{
    return runtime.FuncForPC(reflect.ValueOf(i).Pointer()).Name()
}

for _, test := range []func(*testing.T, string){
    ...
    testSubscriber,
    testSubscription,
}{
    r.Run(getFuncName(test), func(t *testing.T){ test(t, "cloud") })
}
```

Reply:
I think that these changes should be in a different PR. I have made the tests reflect the structure of the existing tests.

Reply:
Yeah that's fair